src/test/unit/services/category.spec.js

```diff
@@ -1,6 +1,6 @@
 const categoryService = require('~/services/category')
 const Category = require('~/models/category')
-const Subject = require('~/models/Subject')
+const Subject = require('~/models/subject')
```

Comment:
By default, GitHub is case-insensitive in filenames.   And I don't see rename if subject file in this pr. Be careful with this.